src/pixbuf.rs

```diff
@@ -244,4 +246,21 @@ impl Pixbuf {
             pixels[pos + 3] = alpha;
         }
     }
+
+    pub fn copy(&self) -> Pixbuf {
+        unsafe {
+            let copy = ffi::gdk_pixbuf_copy(self.to_glib_none().0);
+            FromGlibPtrFull::from_glib_full(copy)
+        }
+    }
+
+    #[cfg(feature = "v2_36")]
+    pub fn copy_options(&self, dest_pixbuf: &mut Pixbuf) -> Result<(), BoolError> {
+        let err: glib_ffi::gboolean = unsafe {
+            let src = self.to_glib_none().0;
```

Comment:
See https://github.com/gtk-rs/glib/blob/master/src/translate.rs#L35-L40
`self.to_glib_none().0` is alive only on time where `self.to_glib_none()` alive,
in end of this line temporary variable, created by `self.to_glib_none()` is dropped and src points on freed memory.
This why this expression need be moved to function call.